aten/src/ATen/native/Normalization.cpp

```diff
@@ -461,7 +461,9 @@ std::tuple<Tensor, Tensor, Tensor> _batch_norm_impl_index_backward(
   const Tensor& save_mean = c10::value_or_else(save_mean_opt, [] {return Tensor();});
   const Tensor& save_var_transform = c10::value_or_else(save_var_transform_opt, [] {return Tensor();});
 
-  if (impl_index == 0) {
+  // backward in inference mode is not supported in cudnn, fallback to native
+  // TODO: verify the same thing in miopen
```

Comment:
Is this TODO for later?

Reply:
Yes. I would need confirmation from someone who know rocm and its libraries to verify that.